config/application.config.php

```diff
@@ -0,0 +1,64 @@
+<?php
+return array(
+    // This should be an array of module namespaces used in the application.
+    'modules' => array(
+        'Application',
+    ),
+
+    // These are various options for the listeners attached to the ModuleManager
+    'module_listener_options' => array(
+        // This should be an array of paths in which modules reside.
+        // If a string key is provided, the listener will consider that a module
+        // namespace, the value of that key the specific path to that module's
+        // Module class.
+        'module_paths' => array(
```

Comment:
Not needed if we use composer autoloading. Use an empty array, reduce the file to a minimum